src/shogun/classifier/AveragedPerceptron.cpp

```diff
@@ -1,15 +1,16 @@
 /*
  * This software is distributed under BSD 3-clause license (see LICENSE file).
  *
- * Authors: Soeren Sonnenburg, Sergey Lisitsyn, Giovanni De Toni, 
+ * Authors: Soeren Sonnenburg, Sergey Lisitsyn, Giovanni De Toni,
```

Comment:
could you pls check your editor and make it not do such newline character changes? They pollute the diff